pages/Messages/Ordering.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Ordering
+route: /Messages/Ordering
+menu: Messages
+---
+
+# Messages Ordering
+
+## Specification Status
+
+| Version | Status |
+---------- | ---------
+| 0.1     | Tentative |
+
+## Purpose
+
+1. Describe a standard for ordering DSNP messages and resolving conflicts in DSNP message with time dependent context.
+
+## Assumptions
+
+* All assumptions from [DSNP Messages](/Messages/Overview)
+
+## Ordering
+
+DSNP messages must be ordered at the highest level by the block number of the block in which a message's batch is announced with higher block numbers indicating more recent messages and lower block numbers representing older ones.
```

Comment:
Throughout this section it would be good to substitute "older" with "sorted first" ("sorted earlier"?) and newer with "sorted last" to emphasize this isn't a strictly temporal search.

Reply:
I didn't want to say first or last since clients may choose to sort content by newest or oldest. It would make sense for a client to display a broadcast feed in reverse chronological order and display a conversation in forward chronological order.

Reply:
The notion of a sort order implies the notion of first and last, and "earlier" or "later" in the sort. It's unfortunate that a lot of our language to describe sorting revolves around time. The point of this comment is that, to the extent possible, it would be good to separate the notion of sort order from temporal order. So "sorted first" or "sorted earlier" is better that "older", since items that appear earlier in the sort aren't necessarily older in any meaningful sense. "Sorted higher" and "sorted lower" might be better still. How the sort order is used to display items should be irrelevant.